src/ODbL-1.0.xml

```diff
@@ -7,9 +7,11 @@
       </crossRefs>
     <text>
       <titleText>
+	     <optional>##</optional>
          <p>ODC Open Database License (ODbL)</p>
       </titleText>
       <optional>
+	     <optional>###</optional>
```

Comment:
I would prefer `<alt name="header2" match="#*" />` here and, with `header1`, above to keep the Markdown out of our canonical HTML.  More on this approach in #429, #623, etc.

Reply:
Good idea, but currently don't have the time to update.  I'll merge in this version, feel free to submit a PR for the next release.

Reply:
> ... feel free to submit a PR for the next release.

Submitted as #661.  Any reason that couldn't land in this release?